extensions-core/stats/src/main/java/io/druid/query/aggregation/teststats/ZtestPostAggregator.java

```diff
@@ -46,79 +48,92 @@
    http://facweb.cs.depaul.edu/sjost/csc423/documents/test-descriptions/indep-z.pdf
 */
 @JsonTypeName("zscore2sample")
-public class ZtestPostAggregator implements PostAggregator {
+public class ZtestPostAggregator implements PostAggregator
+{
   private final String name;
-  private final List<PostAggregator> fields;
+  private final PostAggregator successCount1;
+  private final PostAggregator sample1Size;
+  private final PostAggregator successCount2;
+  private final PostAggregator sample2Size;
+
 
   @JsonCreator
   public ZtestPostAggregator(
       @JsonProperty("name") String name,
-      @JsonProperty("fields") List<PostAggregator> fields
-  ) {
+      @JsonProperty("successCount1") PostAggregator successCount1,
+      @JsonProperty("sample1Size") PostAggregator sample1Size,
+      @JsonProperty("successCount2") PostAggregator successCount2,
+      @JsonProperty("sample2Size") PostAggregator sample2Size
+
```

Comment:
Unnecessary whitespace

Reply:
Do you mean to remove line 67?
I'm using Druid 11 checkstyle.xml and IntelliJ to do reformatting.  And it passes the stylecheck.
According to IntelliJ's interpretation continuation indent is 4.
Just like to make sure I understand what you mean. :-)

Reply:
Do you mean line#67?  Or if there are more unnecessary whitespace you're referring to?
Just like to make sure we are on the same page. :-)

Reply:
Yes, please remove this line.

Reply:
will do! 👍